lib/config/presets/internal/compatibility.ts

```diff
@@ -1,9 +1,11 @@
+import { gettext } from '../../../i18n';
 import type { Preset } from '../types';
 
 export const presets: Record<string, Preset> = {
   additionalBranchPrefix: {
-    description:
-      'Backwards-compatibility preset to restore `additionalBranchPrefix` settings for multiple managers which were removed in Renovate `v25`.',
+    description: gettext(
+      'Backwards-compatibility preset to restore `additionalBranchPrefix` settings for multiple managers which were removed in Renovate `v25`.'
+    ),
```

Comment:
Under context of GNU Gettext, all strings requiring translation should be marked in the sources. Making is done in such a way that each translatble string appears to be the sole argument of some  function or preprocessor macro.  There are only a few such possible functions or macros meant for translation, and their names are said to be `marking keywords`.

And "`gettext`" is a primary marking keyword.  Then  the [`xgettext`](https://www.gnu.org/software/gettext/manual/gettext.html#xgettext-Invocation
) program, which provided by Gettext utilities, can  scan and extract those translatable strings from sources.

So we can only pass a string literal to `gettext`, instead of a variable unfortunately.